README.md

```diff
@@ -44,10 +44,17 @@ Use the YunoHost admin panel to install Friendica by entering the GitHub repo ad
 
 		https://github.com/YunoHost-Apps/friendica_ynh
 
-## User with LDAP admin rights
-**For admin rights**: When installation is complete, you will need to visit your domain page and login with the **admin account username and password** which was entered at the time of installation process. You can then create your profile and access the admin panel.
-
- **For normal YunoHost users :** Normal LDAP users can login through Ldap authentication and create there profiles.
+### First Login
+as explain in the [issue #43](https://github.com/YunoHost-Apps/friendica_ynh/issues/43), 
```

Comment:
Do we need to refer to this issue or the steps in the readme are clear enough?

Reply:
why not ?

but if not, no trouble on my side, it was just for traceability

Reply:
@isAAAc I have no problem with mentioning the issue and I can revert the change if you think it's better to understand the context. My point was that your explanation summarize the issue clearly enough for not having to go thought the issue itself.